src/tasks/LibraryBuilder/LibraryBuilder.cs

```diff
@@ -273,10 +287,26 @@ private static void WriteLinkerScriptFile(string exportsFile, List<string> expor
 
     private void WriteAutoInitializationFromTemplate()
     {
-        File.WriteAllText(Path.Combine(OutputDirectory, "autoinit.c"),
-            Utils.GetEmbeddedResource("autoinit.c")
+        string autoInitialization = Utils.GetEmbeddedResource("autoinit.c")
                 .Replace("%ASSEMBLIES_LOCATION%", !string.IsNullOrEmpty(AssembliesLocation) ? AssembliesLocation : "DOTNET_LIBRARY_ASSEMBLY_PATH")
-                .Replace("%RUNTIME_IDENTIFIER%", RuntimeIdentifier));
+                .Replace("%RUNTIME_IDENTIFIER%", RuntimeIdentifier);
+
+        if (BundlesResources)
+        {
+            var bundledRuntimeConfigInfo = BundledRuntimeConfig.First();
```

Comment:
I thikn we should do a LogAsErrorException in case where there is more than one BundledRuntimeConfig passed in instead of "silently" just use the first one in the list.